Sources/PackageModel/Module.swift

```diff
@@ -129,9 +131,25 @@ public class ClangModule: CModule {
     }
 }
 
+/// A dictionary containing mapping of C family source extension to their file type in Xcode.
+private let xcodeCLanguageFileTypeDict = [
+    ".c": "sourcecode.c.c",
+    ".m": "sourcecode.c.objc",
+    ".cxx": "sourcecode.cpp.cpp",
+    ".cc": "sourcecode.cpp.cpp",
+    ".cpp": "sourcecode.cpp.cpp",
+    ".mm": "sourcecode.cpp.objcpp",
+]
```

Comment:
Should we get an enum for the suffixes we support and make sure this is shared with the C language target support? I wouldn't want them to get out of sync.